Platform/TigerlakeBoardPkg/CfgData/CfgData_Silicon.yaml

```diff
@@ -294,7 +294,130 @@
                      Enable/disable HD Audio DMIC_N.
       length       : 0x02
       value        : { 0x00, 0x00 }
+  - Usb20Enable :
+    - $STRUCT      :
+        name         : Usb20Enable
+        struct       : USB20_PORT_EN_DATA
+        length       : 0x04
+        value        : 0x0000FFFF
+    - Port0         :
+        name         : Enable USB2 port0
+        type         : Combo
+        option       : $EN_DIS
+        help         : >
+                       Enable/disable per USB2 port
+        length       : 1b
+    - Port1         :

```

Comment:
If you want to avoid duplicating things,  the easier approach is to use template.
Here is an example:
1. Define a new template in CfgDataDef.yaml as below:
  USB_PORT_TMPL: >
    - Usb$(1)Port$(2) :
          name         : Enable USB port $(2)
          type         : Combo
          option       : $EN_DIS
          help         : >
                         Enable/disable USB port.
          length       : 1b
          value        : 1

2.  Add following into Cfgdata_Silicon.yaml
  - PortUsb20Enable :
      - $ACTION:
          page         : SILUSB2:SIL:"USB Ports"
      - $STRUCT      :
          name         : Enable USB2 Ports
          length       : 0x02
      - !expand { USB_PORT_TMPL : [ 2, 0 ] }
      - !expand { USB_PORT_TMPL : [ 2, 1 ] }
      - !expand { USB_PORT_TMPL : [ 2, 2 ] }
      - !expand { USB_PORT_TMPL : [ 2, 3 ] }
      - !expand { USB_PORT_TMPL : [ 2, 4 ] }
      - !expand { USB_PORT_TMPL : [ 2, 5 ] }
      - !expand { USB_PORT_TMPL : [ 2, 6 ] }
      - !expand { USB_PORT_TMPL : [ 2, 6 ] }
      - !expand { USB_PORT_TMPL : [ 2, 7 ] }
      - !expand { USB_PORT_TMPL : [ 2, 8 ] }
      - !expand { USB_PORT_TMPL : [ 2, 9 ] }
      - Reserved:
          name         : Reserved
          type         : Reserved
          length       : 6b
3.  Now you can change default with DLT:
    EX:    SILICON_CFG_DATA.PortUsb20Enable.Usb2Port1                  | 1